sym_iwram.txt

```diff
@@ -2,6 +2,7 @@
 build/sa2/src/lib/m4a.o(.bss.code);
 build/sa2/src/multi_boot.o(.bss);
 /* space 0x20 */
+. = 0x00000408; gUnknown_03000408 = .; . += 2;
```

Comment:
Are you sure there are globals here? All game related globals should start after core

Reply:
This looks more to me like an iwram offset so probably a task member at offset 0x408

Reply:
IMO https://decomp.me/scratch/UPpy9

Reply:
I don't know, it doesn'T make sense for me right now... but also the code doesn't match yet, so maybe?

Reply:
It's very unlikely to be referencing a global var which relates to anything there. IMO this is probably a mistake with a null pointer? Or at least a pointer to `IWRAM_START` instead

Reply:
I would definitely say remove the global until we are sure what's going on. It's not even referenced anywhere else so not really any reason to believe it's absolutely a global

Reply:
Alright!
Really wondering what this is about!